tests/30rooms/60version_upgrade.pl

```diff
@@ -59,6 +61,45 @@ sub upgrade_room {
    });
 }
 
+=head2 is_direct_room
+
+    is_direct_room( $user, $room_id )->then( sub {
+        my ( $is_direct ) = @_;
+    })
+
+Check if a room ID is considered direct by the given user.
+
+=cut
+
+sub is_direct_room {
+   my ( $user, $room_id ) = @_;
+
+   # Download account data events from sync
+   matrix_sync( $user )->then( sub {
```

Comment:
How about using `GET /_matrix/client/r0/user/{user}/account_data/m.direct` (https://github.com/matrix-org/matrix-doc/issues/1339) ?

Reply:
Ah cool.

Reply:
Indeed this is already a function: https://github.com/matrix-org/sytest/blob/665210b5e13e4adf9194512d44677a2a9ef7ed0a/tests/10apidoc/44account_data.pl#L71-L79